gradle.properties

```diff
@@ -11,3 +11,5 @@ org.gradle.jvmargs=-Xmx6096M
 
 # hint by https://docs.gradle.org/current/userguide/performance.html#enable_the_build_cache
 org.gradle.caching=true
+org.gradle.parallel=true
+org.gradle.daemon=true
```

Comment:
Are you sure abot this?

We should have parallel tests: https://github.com/JabRef/jabref/pull/9797

---

Please remove this - we do not know about any side effects.